script/openqa-clone-custom-git-refspec

```diff
@@ -21,22 +21,19 @@ Examples:
  openqa-clone-custom-git-refspec -n -c '--show-progress' https://github.com/coolgw/os-autoinst-distri-opensuse/tree/nfs https://openqa.opensuse.org/tests/835060 DESKTOP=textmode
  openqa-clone-custom-git-refspec https://github.com/foursixnine/os-autoinst-distri-opensuse/tree/oopsitsbrokenagain https://openqa.opensuse.org/tests/3128467 NEEDLES_DIR='https://github.com/foursixnine/os-autoinst-needles-opensuse.git#oopsitsbrokenagain'
 
-You need to set the environment variable GITHUB_TOKEN to use authenticated requests.
+To use authenticated requests, you need to either provide a GitHub token via a configuration file at '~/.config/openqa/github-token.conf' or set
+the GITHUB_TOKEN environment variable.
+
+Use this snippet to do so:
+
+ GH_ACCESS_TOKEN="YourTokenHere12345"
```

Comment:
This probably needs an `export` or should clarify that it is about the config file. However, I suggest to leave the "snippet" out completely as people probably know how they want to set this environment variable for their purposes and with my other suggestion the use of the config file should be clear enough.

```suggestion
```

Reply:
This snippet refers to the sentence above. I have changed the wording to make it more clear that the snippet goes into the `github-token.conf`